functions/pupil-prefs/index.js

```diff
@@ -1,19 +1,97 @@
 'use strict'
 
-const uuid = require('uuid/v4')
 const moment = require('moment')
+const sqlService = require('less-tedious')
+const { TYPES } = require('tedious')
+const uuid = require('uuid/v4')
+const winston = require('winston')
+
+const config = require('../config')
+const accessArrangementsTable = '[accessArrangements]'
+const checkTable = '[check]'
+const pupilAccessArrangementsTable = '[pupilAccessArrangements]'
+const pupilFontSizesTable = '[pupilFontSizes]'
+const pupilColourContrastsTable = '[pupilColourContrasts]'
+const pupilTable = '[pupil]'
+const schema = '[mtc_admin]'
 
-module.exports = function (context, prefsMessage) {
-  context.log('pupil-preferences message received', prefsMessage.checkCode)
+winston.level = 'error'

```

Comment:
why are we hard coding the log level here?  any particular reason?

Reply:
@GuyHarwood it's removed now as it is not serving any real purpose.